Option to typeset exercises one-per-page

eamsden opened the pull request:
Adds a new options file opt-exerciseperpage.tex, two new top-level tex files `hott-a4-exercises.tex` and `hott-letter-exercises.tex` and corresponding makefile targets. The resulting PDFs swill start each exercise on a new page.

This is useful for printing exercises in a form that affords space to work each exercise. Example exercise page:
[hott-letter-exercises-1.1.pdf](https://github.com/HoTT/book/files/1864904/hott-letter-exercises-1.1.pdf)

mikeshulman commented on README.md:
```diff
@@ -34,8 +34,10 @@ code. Also, the file `version.tex` is generated on the fly, so you will need the
+*  make hott-letter-exercises.pdf -- the book in black & white, letter paper format, but with exercises one-per-page
 * `make hott-a4.pdf hott-a4.pdf` -- the book in black & white, A4 paper format,
    for printing at home, as well as a color cover (just two pages)
+*  make hott-letter-exercises.pdf -- the book in black & white, A4 paper format, but with exercises one-per-page
```
This should probably be `make hott-a4-exercises.pdf`

mikeshulman commented on hott-a4-exercises.tex:
```diff
@@ -0,0 +1,31 @@
+% -*- mode: LaTeX -*-
+% This is a main LaTeX file for generating the HoTT book
+% for printing on A4 paper. The links are black in this
+% version.
```
Do you want to add something to the comment mentioning the one-per-page aspect of this file?

mikeshulman commented on hott-letter-exercises.tex:
```diff
@@ -0,0 +1,31 @@
+% -*- mode: LaTeX -*-
+% This is a main LaTeX file for generating the HoTT book
+% for printing on letter paper. The links are black in this
+% version.
```
Do you want to add something to the comment mentioning the one-per-page aspect of this file?

eamsden replied:
>  Is the idea to use these as group-work exercises in a class or something?
I think it would be helpful for that.
My motivation was for myself, working through the exercises, and I hoped it might be helpful to others.

eamsden commented:
@mikeshulman  I pushed a new commit addressing the comments in your review. I did at first attempt to generate a file containing only the exercises, but there are a lot of `\ref`s from the exercises to the main book which broke in that case. My TeX foo is not quite up to fixing that! It seems reasonable to simply generate the whole book, and then print only the exercise pages that one is interested in working.

My initial thought was to find a PDF viewer that had a `print selection` feature and use that to print exercises from the original book, but this turned out to be a path of lower resistance.

mikeshulman commented on hott-a4-exercises.tex:
```diff
@@ -1,7 +1,7 @@
 % This is a main LaTeX file for generating the HoTT book
-% for printing on A4 paper. The links are black in this
-% version.
+% for printing on A4 paper, with exercises on per page.
```
Another typo: "on per page" should be "one per page", or maybe "printed one per page" to match what you wrote it `hott-letter-exercises`.

eamsden replied:
Fixed in 9386c1e
